### PR TITLE
Adds Ukranian locale and updates locales

### PR DESCRIPTION
**Pull request by mouse-reeve**

### CODE REVIEW

**Comment by 0x29a** [comment]
Oof, thank you for adding the locale, @mouse-reeve. May I ask you to wait a few days before merging this, until I finish translating at least `django.po`? (Nit: the title is missing one letter: `Ukranian` -> `Ukrainian`). I'll hurry up. :slightly_smiling_face:

File: bookwyrm/settings.py [hunk 1]
```diff
@@ -328,6 +328,7 @@
     ("pt-pt", _("Português Europeu (European Portuguese)")),
     ("ro-ro", _("Română (Romanian)")),
     ("sv-se", _("Svenska (Swedish)")),
+    ("uk-ua", _("Українська (Ukranian)")),
```
Comment (0x29a) [inline bookwyrm/settings.py:331]:
```suggestion
    ("uk-ua", _("Українська (Ukrainian)")),
```

File: bw-dev [hunk 2]
```diff
@@ -150,6 +150,7 @@ case "$CMD" in
         git fetch origin l10n_main:l10n_main
         git checkout l10n_main locale/ca_ES
         git checkout l10n_main locale/de_DE
+        git checkout l10n_main locale/en_UK
```
Comment (0x29a) [inline bw-dev:153]: Shouldn't this be rather `uk_UA`?

File: bookwyrm/migrations/0189_alter_user_preferred_language.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                    ("fr-fr", "Français (French)"),
+                    ("lt-lt", "Lietuvių (Lithuanian)"),
+                    ("nl-nl", "Nederlands (Dutch)"),
+                    ("no-no", "Norsk (Norwegian)"),
+                    ("pl-pl", "Polski (Polish)"),
+                    ("pt-br", "Português do Brasil (Brazilian Portuguese)"),
+                    ("pt-pt", "Português Europeu (European Portuguese)"),
+                    ("ro-ro", "Română (Romanian)"),
+                    ("sv-se", "Svenska (Swedish)"),
+                    ("uk-ua", "Українська (Ukranian)"),
```
Comment (0x29a) [inline bookwyrm/migrations/0189_alter_user_preferred_language.py:37]:
```suggestion
                    ("uk-ua", "Українська (Ukrainian)"),
```

File: bw-dev (hunk 2 above)
Comment (uandreew) [reply bw-dev:153]: > Shouldn't this be rather `uk_UA`?

yep

Comment (mouse-reeve) [reply bw-dev:153]: Oops! Thank you for catching that

**Comment by mouse-reeve** [comment]
> Oof, thank you for adding the locale, @mouse-reeve. May I ask you to wait a few days before merging this, until I finish translating at least `django.po`? (Nit: the title is missing one letter: `Ukranian` -> `Ukrainian`). I'll hurry up. 🙂

Sure thing! Thank you so much for updating the translations

**Comment by mouse-reeve** [comment]
I was able to replicate these tests failing; when I changed my system clock to a different day, the tests pass.

**Comment by 0x29a** [comment]
@mouse-reeve, I'll try to look into this today. There are also still 3 pages left to translate, I apologize for the delay.

**Pull request merged**